tools/interactive/interactivetool_askomics.xml

```diff
@@ -1,26 +1,44 @@
-<tool id="interactive_tool_askomics" tool_type="interactive" name="AskOmics" version="0.1">
+<tool id="interactive_tool_askomics" tool_type="interactive" name="AskOmics" version="1.0">
     <description>a visual SPARQL query builder</description>
     <requirements>
-        <container type="docker">quay.io/askomics/askomics-ie:17.12_g19.09</container>
+        <container type="docker">askomics/flaskomics-with-dependencies:3.1.1</container>
     </requirements>
     <entry_points>
         <entry_point name="AskOmics instance on $infile.display_name" requires_domain="True">
-            <port>6543</port>
-            <url>/login_api_gie?key=abcd</url>
+            <port>5000</port>
+            <url>/loginapikey/${__user_name__}</url>
         </entry_point>
     </entry_points>
     <environment_variables>
+        <!-- AskOmics user -->
+        <environment_variable name="CREATE_USER">true</environment_variable>
+        <environment_variable name="USER_FIRST_NAME">${__user_name__}</environment_variable>
+        <environment_variable name="USER_LAST_NAME">Galaxy</environment_variable>
+        <environment_variable name="USER_USERNAME">${__user_name__}</environment_variable>
+        <environment_variable name="USER_EMAIL">${__user_email__}</environment_variable>
+        <environment_variable name="USER_APIKEY">${__user_name__}</environment_variable>
+        <!-- Galaxy info -->
         <environment_variable name="GALAXY_URL">$__galaxy_url__</environment_variable>
         <environment_variable name="API_KEY" inject="api_key" />
```

Comment:
This one should be `GALAXY_API_KEY` instead of `API_KEY` (like https://github.com/usegalaxy-eu/galaxy/pull/42)

Reply:
Do you know what is the `USER_APIKEY` above?